lib/cloud_controller/paging/sequel_paginator.rb

```diff
@@ -25,7 +25,10 @@ def get_page(dataset, pagination_options)
     end
 
     def can_paginate_with_window_function?(dataset)
-      dataset.supports_window_functions? && (!dataset.opts[:distinct] || !dataset.requires_unique_column_names_in_subquery_select_list?)
+      enable_paginate_window = true if Config.config.get(:db, :enable_paginate_window).nil?
+      enable_paginate_window = Config.config.get(:db, :enable_paginate_window) unless Config.config.get(:db, :enable_paginate_window).nil?
```

Comment:
Proposal for a shorter and more concise reading of the config value:
```
enable_paginate_window = Config.config.get(:db, :enable_paginate_window).nil? ? true : Config.config.get(:db, :enable_paginate_window)
```

Reply:
Fixed